[Key Transparency] Add tile (#17048)
cloudflare/cloudflare-docs: src/content/docs/key-transparency/api/namespaces.mdx

---
pcx_content_type: overview
title: Namespaces
sidebar:
  order: 2
---

> The Cloudflare Key Transparency API is organized in namespaces, representing a log Cloudflare Auditor monitors. If you want to register a namespace, contact us.

Reviewer: ```suggestion
The Cloudflare Key Transparency API is organized in namespaces, representing log Cloudflare Auditor monitors. If you want to register a namespace, contact us.
```

I'm not sure what `log Cloudflare Auditor monitors` are, so there's a chance this grammar change is not applicable.

> The Cloudflare Key Transparency API is organized in namespaces, each one representing a Log Cloudflare Auditor monitors. If you want to register a namespace, contact us.

Author: ```suggestion
The Cloudflare Key Transparency API is organized in namespaces, each one representing a Log monitored by Cloudflare Auditor. If you want to register a namespace, contact us.
```

## Create a namespace

The following fields are required when making a `POST` request:

- `name`
- `public`
- `root`
- `signature_version`: 

>   - 0x0001 is for [Protobuf serialisation](https://github.com/cloudflare/plexi/blob/main/plexi_core/src/proto/specs/types.proto) Ed25519 signature from the Auditor 

Reviewer: ```suggestion
  - 0x000 for [Protobuf serialisation](https://github.com/cloudflare/plexi/blob/main/plexi_core/src/proto/specs/types.proto) Ed25519 signature from the Auditor 
```

>   - 0x0001 is for [Protobuf serialisation](https://github.com/cloudflare/plexi/blob/main/plexi_core/src/proto/specs/types.proto) Ed25519 signature from the Auditor 

Reviewer: ```suggestion
  - 0x0001 for [Protobuf serialisation](https://github.com/cloudflare/plexi/blob/main/plexi_core/src/proto/specs/types.proto) Ed25519 signature from the Auditor 
```

Reviewer 2: resolved

>   - 0x0002 is for [bincode serialisation](https://github.com/bincode-org/bincode/blob/trunk/docs/spec.md) E25519 serialisation by the Auditor

Reviewer: ```suggestion
  - 0x0002 for [bincode serialisation](https://github.com/bincode-org/bincode/blob/trunk/docs/spec.md) E25519 serialisation by the Auditor
```

Reviewer 2: resoved
